net-wireless/wavemon/files/wavemon-0.9.3-build.patch

```diff
@@ -0,0 +1,40 @@
+--- a/configure.ac	2020-12-26 17:44:01.616731241 +0100
++++ b/configure.ac	2020-12-26 17:45:02.447292151 +0100
+@@ -22,6 +22,8 @@
+ AC_PROG_CPP
+ AC_PROG_INSTALL
+ 
++PKG_PROG_PKG_CONFIG()
++
+ # Checks for header files.
+ AC_HEADER_STDC
+ AC_HEADER_TIME
+@@ -29,6 +31,7 @@
+ 		  net/if_arp.h netinet/ether.h net/ethernet.h pthread.h],
+ 		 [], [AC_MSG_ERROR($ac_header not found)])
+ AC_CHECK_HEADERS([ncursesw/curses.h])
++PKG_CHECK_MODULES(ncurses,ncurses,LIBS="$LIBS $ncurses_LIBS",)
+ 
+ # linux/if.h needs 'struct sockaddr'
+ # linux/wireless.h in turn depends on linux/if.h
+--- a/Makefile.in	2017-07-14 21:35:18.154423280 +0300
++++ b/Makefile.in	2017-07-14 21:35:34.613402738 +0300
+@@ -9,7 +9,7 @@
+ install-suid-root: exec_perms = 4755
+ 
+ CC	 ?= @CC@
+-CFLAGS	 ?= @CFLAGS@ @LIBNL3_CFLAGS@
++CFLAGS	 = @CFLAGS@ @LIBNL3_CFLAGS@
```

Comment:
However looks like this patch isn't working as you wish,
```
 * QA Notice: Files built without respecting CFLAGS have been detected
 *  Please include the following list of files in your report:
 * /usr/bin/wavemon
```

Reply:
I removed the CFLAGS from `configure.ac`, so now the CFLAGS are respected.